ui-v2/app/services/session.js

```diff
@@ -9,6 +9,7 @@ export default Service.extend({
       dc: dc,
     });
   },
+  // TODO: Why Key? Probably should be findBySlug like the others
   findByKey: function(slug, dc) {
     return get(this, 'store').queryRecord('session', {
       id: slug,
```

Comment:
Again a little note for later. Some of the names of these don't quite follow the same naming system, I'm sure there was a reason for that when I did it. But once this is all complete I'll probably review the names and make sure they follow the same naming pattern - and if they don't - that they have a reason for it.